modules/helm-filtering-helm-charts-by-certification-level.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/configuring-custom-helm-chart-repositories.adoc
+
+[id="filtering-helm-charts-by-certification-level_{context}"]
+= Filtering Helm Charts by their certification level
+
+As a user, you can use the source filter to filter the Helm charts based on their certification level in the *Developer Catalog*.
+
+.Procedure
+
+. In the *Developer perspective*, navigate to the *+Add* view and select a project. From the *Developer Catalog*, select the *Helm Chart* option to see all the Helm charts in the *Developer Catalog*.
```

Comment:
This should be the second step - From the *Developer Catalog*, select the *Helm Chart* option to see all the Helm charts in the *Developer Catalog*.

Reply:
Done!